README.md

```diff
@@ -9,11 +9,11 @@
 The service-catalog project is in incubation to bring integration with service
 brokers to the Kubernetes ecosystem via the [Open Service Broker
 API](https://github.com/openservicebrokerapi/servicebroker). A service broker
-is an endpoint that manages a set of services.  The end-goal of the service-
-catalog project is to provide a way for Kubernetes users to consume services
-from brokers and easily configure their applications to use those services,
-without needing detailed knowledge about how those services are created /
-managed.
+is an endpoint that manages a set of capabilities (called services).  The end-
```

Comment:
"capabilities called services" seems very strange to me. Services aren't "capabilities"; capability is the _power/ability_ to do something, while services are external resources that provide clients their capabilities.

I would just substitute "capabilities" with "resources".

Reply:
How about 'managed software offerings called services', which is how the spec defines the term.

Reply:
That sounds better to me. We already say it "manages" them, though, so I think just "software offerings called services" is good.